Improve docs for release process (#70)
capactio/website: community/release.md

# Capact release process

This document describes Capact release process.

---
**NOTE**

Releasing new Capact version requires admin privileges in the `capact`, `hub-manifests` and `website` repositories.

> ---

Reviewer: 1. Please revert the changes to the header:

```markdown
---
sidebar_position: 6
---

# Release process

This document describes Capact release process.
```

2. Also, we can easily convert the NOTE to one of prerequisites ("Prerequisites" section). Let's remove it from here and apply the suggestion below 🙂

## Prerequisites

> - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
> - An GitHub Environment named `Release` on [`capactio/capact`](https://github.com/capactio/capact) repository with the following secrets set:

Reviewer: ```suggestion
- Admin role on the `capact`, `hub-manifests` and `website` repositories.
- Disabled branch protection rules for all repositories. This allows the release jobs to commit directly to `main` and release branches. 

    > **NOTE:** Do not forget to re-enable them after creating a release.

- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
- An GitHub Environment named `Release` on [`capactio/capact`](https://github.com/capactio/capact) repository with the following secrets set:
```

>   - `GCS_CREDS` — Base64 encoded Google Cloud Platform credentials in JSON format to access Google Cloud Storage for binary and chart releases. Make sure that it has `storage.objects.create` permission,
>   - `GH_PAT` — GitHub personal access token. Make sure that it has selected `repo` and `write:packages` scopes.
> 
> All our repositories have enabled branch protection rules which disallow directly committing to the repository. Make sure to disable them before proceeding and re-enable them after making a release.

Reviewer: ```suggestion
    - `GCS_CREDS` — Base64-encoded Google Cloud Platform credentials in JSON format to access Google Cloud Storage for binary and chart releases. Make sure that it has `storage.objects.create` permission.
    - `GH_PAT` — GitHub personal access token. Make sure that it has selected `repo` and `write:packages` scopes.
```

## Prerequisites

- Admin role on the `capact`, `hub-manifests` and `website` repositories.
- Disabled branch protection rules for all repositories. This allows the release jobs to commit directly to `main` and release branches.

    > **NOTE:** Do not forget to re-enable them after creating a release.

> - An GitHub Environment named `Release` on [`capactio/capact`](https://github.com/capactio/capact) repository with the following secrets set:

Reviewer: ```suggestion
- An GitHub Environment named `Release` on [`capactio/hub-manifests`](https://github.com/capactio/hub-manifests) and [`capactio/website`](https://github.com/capactio/website) repositories with the following secret set:
```

    - `GCS_CREDS` — Base64-encoded Google Cloud Platform credentials in JSON format to access Google Cloud Storage for binary and chart releases. Make sure that it has `storage.objects.create` permission.

>     - `GH_PAT` — GitHub personal access token. Make sure that it has selected `repo` and `write:packages` scopes.
> - An GitHub Environment named `Release` on [`capactio/hub-manifests`](https://github.com/capactio/hub-manifests) and [`capactio/website`](https://github.com/capactio/website) repositories with the following secret set:
>     - `GH_PAT` — GitHub personal access token. Make sure that it has selected `repo` and `write:packages` scopes. Must be in format: `"<username>:<PAT>"`

Reviewer: ```suggestion
    - `GH_PAT` — GitHub personal access token. Make sure that it has selected `repo` and `write:packages` scopes. Must be in format: `"<username>:<PAT>"`
```
